controllers/disruption_rollout_controller.go

```diff
@@ -32,6 +35,18 @@ func (r *DisruptionRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	instance := &chaosv1beta1.DisruptionRollout{}
 	randSource := rand.New(rand.NewSource(time.Now().UnixNano()))
 
+	// reconcile metrics
+	r.handleMetricSinkError(r.MetricsSink.MetricReconcile([]string{"rolloutName:" + instance.Name, "namespace:", instance.Namespace}))
+
+	defer func(tsStart time.Time) {
```

Comment:
what does this func do?

Reply:
This is the function for calculating reconcile duration, but I missed this, it needs to have tags associated with it to differentiate between the functions, thanks for bringing this to my attention.